lib/pfconfig/manager.pm

```diff
@@ -189,9 +189,8 @@ sub touch_cache {
         close($fh);
     }
     if ( -e $filename ) {
-        sysopen( my $fh, $filename, O_RDWR | O_CREAT );
-        POSIX::2008::futimens( fileno $fh );
-        close($fh);
+        my $command = 'touch --date=@'.time.' '."'".$filename."'";
+        `$command`;
```

Comment:
The old way with POSIX doesn't seem to use the same source as Time::HiRes as it was nanoseconds late of the latest.

That meant the processes were being expired a few nanoseconds after which made unit tests fail.

I'll try utime and see if it works with subseconds